osschallenge/views.py

```diff
@@ -112,34 +120,25 @@ def MyTaskIndexView(request, username):
     template_name = 'osschallenge/mytasksindex.html'
     current_user_id = request.user.id
     user_task_objects = Task.objects.filter(assignee_id=current_user_id)
-    user_task_list = []
-    for obj in user_task_objects:
-        user_task_list.append(obj)
-    for task in user_task_objects:
+    try:
+        matches = user_task_objects.filter(
+            Q(title__icontains=request.GET.get('search')) |
+            Q(project__title__icontains=request.GET.get('search'))
+        ).distinct().order_by('id')
+    except ValueError:
```

Comment:
What is throwing a `ValueError` here? When `request.GET` is Null?

Reply:
It throws a ValueError if request.GET is None.

Reply:
It might be a bit cleaner to write it like this then:
```python
search = request.GET.get('search') if request.GET else None
if search:
   (...)
```
```

Reply:
I changed this.